tracing-subscriber/src/filter/env/directive.rs

```diff
@@ -9,10 +9,10 @@ use tracing_core::{span, Level, Metadata};
 // TODO(eliza): add a builder for programmatically constructing directives?
 #[derive(Debug, Eq, PartialEq)]
 pub struct Directive {
-    target: Option<String>,
```

Comment:
Why the change in visibility?

Reply:
Otherwise it won't compile:

```
error[E0616]: field `target` of struct `filter::env::directive::Directive` is private
   --> tracing-subscriber/src/filter/env/mod.rs:258:63
    |
258 |                 let target = if let Some(target) = &directive.target {
    |                                                               ^^^^^^ private field
```

Reply:
Ah, fair. In that case, making this `pub(crate)` is reasonable to me.

From a (very!) pedantic style/code formatting point of view, it'd be nice to have the pub(crate) fields grouped together at the end of the struct.